tests/phpunit/api/v3/CustomGroupTest.php

```diff
@@ -300,11 +300,19 @@ public function testCustomGroupCreateGroup() {
       'help_post' => 'This is Post Help For Test Group 8',
     );
 
-    $customGroup = $this->callAPISuccess('custom_group', 'create', $params);
+    $customGroup = $this->callAPISuccess('CustomGroup', 'create', $params);
```

Comment:
These ought to be equivalent. Any reason why this change was made?

Reply:
My understanding is that our preferred format for the entity is the camel case so it was for that reason